makedmdconf.d

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env rdmd
+module makedmdconf;
+
+import core.stdc.stdlib : exit;
+import std.getopt, std.string;
+import std.path, std.file, std.stdio;
+import std.process;
+
+int main(string[] args)
+{
+    bool mscoff = false;
+
+    auto helpInfo = getopt(args,
+        "mscoff", &mscoff);
+    enum nonOptionArgCount = 3;
+    const wrongArgCount = (args.length != nonOptionArgCount + 1);
```

Comment:
Better make all of these required arguments flags, then the user gets a better error message and you can let getopt do its job.
This will also help when in the future more flags get added (or removed).